masterlist.yaml

```diff
@@ -309,7 +309,7 @@ globals:
         text: 'Din installerede version af Place Everywhere er ikke kompatibel med din version af Fallout 4.'
       - lang: ja
         text: 'インストールされているPlace Everywhereは使用中のバージョンのFallout 4との互換性がありません。'
-    condition: '(version("F4SE/Plugins/place.dll", "1.18.0.1111", >=) and not version("../Fallout4.exe", "1.10.111.0", ==)) or (version("F4SE/Plugins/place.dll", "1.18.0.1111", <) and version("../Fallout4.exe", "1.10.111.0", ==))'
+    condition: '(file("F4SE/Plugins/place.dll") and version("F4SE/Plugins/place.dll", "1.18.0.1111", >=) and not version("../Fallout4.exe", "1.10.111.0", ==)) or (file("F4SE/Plugins/place.dll") and version("F4SE/Plugins/place.dll", "1.18.0.1111", <) and version("../Fallout4.exe", "1.10.111.0", ==))'
```

Comment:
As Elminster said on Discord, this first `file` condition is redundant, because the `version(_, _, >=)` will always be false for a missing file.

Reply:
A missing file evaluates to 0, which is false, is that right?

Reply:
Not exactly. The logic is:

```
if (!fileExists(filePath)) {
  return comparator == "!=" || comparator == "<" || comparator == "<=";
}
```

Only if the file exists does the function go on to compare versions.